README.md

```diff
@@ -57,6 +70,9 @@ Command line interface
 ----------------------
 
 You can control the game from the command line using the rose-admin tool.
+	
+    make admin
```

Comment:
Same issue, a server address is needed, and then a command (e.g. start) with optional arguments (e.g set-rate 5)